src/js/components/purchaseOrder/PurchaseOrderListTable.jsx

```diff
@@ -271,50 +312,55 @@ const PurchaseOrderListTable = ({
       className: 'active-circle',
       headerClassName: 'header',
       fixed: 'left',
-      maxWidth: 100,
-      Cell: row => (<PurchaseOrderStatus status={row.original.status} />),
+      width: 160,
+      tooltip: true,
+      Cell: row => (
+        <TableCell {...row} tooltipLabel={getStatusTooltip(row.original.status)}>
```

Comment:
Actually you could do the same with the `link` above, as you did here passing `tooltipLabel`